src/py/aspen/config/config.py

```diff
@@ -29,7 +30,16 @@ def DEBUG(self):
 
     @property
     def SECRET_KEY(self):
-        return os.environ.get("SECRET_KEY")
+        # TODO: This probably needs to be sourced from secrets.  Somehow.  Who knows.
```

Comment:
yeah that's a good idea to store in secrets, but then we would need an auth0 secret and an app secret, unless we create a mega secret. this definitely should be more secure for a production deploy

Reply:
yeah, let me fix this.  I don't think we should be committing this as is.

Reply:
i think we can keep the env variable option for now, or maybe have a randomly generated string?

Reply:
ok i'm setting it to a random value for now.  I'll clean this up in a subsequent PR.